src/app/helptext/system/ca.ts

```diff
@@ -24,6 +24,23 @@ export const helptext_system_ca = {
       validation: [Validators.required]
     },
 
+    key_type: {
+      placeholder: T("Key Type"),
+      tooltip: T(
+        'See <a href="https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa"></a>\
+ for more information about key types.'
+      ),
+      validation: [Validators.required]
+    },
+
+    ec_curve: {
+      placeholder: T("EC Curve"),
+      tooltip: T(
+        'Brainpool* curves can be more secure, while secp* curves can be faster. See\
```

Comment:
Remove the stars, this is likely to only confuse the reader rather than tell them to pattern match.
```suggestion
        'Brainpool curves can be more secure, while secp curves can be faster. See\
```